core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -120,23 +128,27 @@ export class TranslateTextService {
     );
   }
 
+  // The following property is set to null
```

Comment:
Like my other comments above -- please explain more about the circumstances in which this case can occur. It is not sufficient to say "there is no X"; that just raises the question of "why is there no X / in what circumstances would the 'no X' case happen".